### PR TITLE
C# Assignment 1 Completed.

### PR DESCRIPTION
**Pull request by emtec-damodhar-jadhao**
C# Practice Assignment 1 Completed and Uploaded.

### CODE REVIEW

File: Assignment_1/Question_2.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                {
+                    at += 1;
+                }else if (E[i]=='.'){
+                    dot += 1;
+                }
+                
+            }
+            if(at>0 && dot > 0)
+            {
+                Console.WriteLine("Email is valid");
```
Comment (nitish-poojari) [inline Assignment_1/Question_2.cs:26]: this will fail if there are more than 2 @... For example ab.c@gmai@l.com

File: Assignment_1/question_4.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
+﻿using System;
+namespace Assignment_1
+{
+    internal class question_4
```
Comment (nitish-poojari) [inline Assignment_1/question_4.cs:4]: use proper class name

File: Assignment_1/question_4.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    {
+        string name = "null";
```
Comment (nitish-poojari) [inline Assignment_1/question_4.cs]: use proper class name

File: Assignment_1/Question_2.cs (hunk 1 above)
Comment (emtec-damodhar-jadhao) [reply Assignment_1/Question_2.cs:26]: Updated code for @ symbol

File: Assignment_1/question_4.cs (hunk 3 above)
Comment (emtec-damodhar-jadhao) [reply Assignment_1/question_4.cs]: updated proper naming for class

**Review by nitish-poojari — APPROVED** [review]

**Pull request merged**